README.md

```diff
@@ -521,18 +507,10 @@ describe AuthenticateUser do
         allow(User).to receive(:authenticate).with("john@example.com", "secret").and_return(nil)
       end
 
-      it "fails" do
-        interactor.call
-
-        expect(context).to be_a_failure
-      end
+      it { is_expected.to be_a_failure }
 
       it "provides a failure message" do
-        expect {
-          interactor.call
-        }.to change {
-          context.message
-        }.from(nil).to be_present
+        expect(subject.message).to_not be_present
```

Comment:
If I'm reading this correctly the original expectation is that the messages goes from nil to being set after the subject has been invoked.  The new expectation asserts that message isn't set after the subject has been invoked.  That doesn't line up with the name of the test or what it is replacing.